tests/cross_functional/system_test/test_cluster_wide_key_rotation_systemtest.py

```diff
@@ -0,0 +1,148 @@
+import logging
+import pytest
+
+from ocs_ci.framework.pytest_customization.marks import (
+    system_test,
+    magenta_squad,
+    ignore_leftovers,
+)
+from ocs_ci.framework.testlib import E2ETest
+from ocs_ci.helpers.e2e_helpers import (
+    Run_fio_till_cluster_full,
+)
+from ocs_ci.helpers.keyrotation_helper import (
+    verify_key_rotation_time,
+    enable_key_rotation,
+    verify_new_key_after_rotation,
+    OSDKeyrotation,
+)
+from ocs_ci.helpers.osd_resize import basic_resize_osd, get_storage_size
+from ocs_ci.helpers.sanity_helpers import Sanity
+from ocs_ci.ocs import warp
+
+log = logging.getLogger(__name__)
+
+
+@magenta_squad
+@system_test
+@ignore_leftovers
+class TestKeyRotationWithClusterFull(E2ETest):
+    @pytest.fixture(autouse=True)
+    def init_sanity(self):
+        """
+        Initialize Sanity instance
+
+        """
+        self.sanity_helpers = Sanity()
+        self.sanity_helpers.health_check()
+
+    @pytest.fixture()
+    def warps3(self, request):
+        warps3 = warp.Warp()
+        warps3.create_resource_warp(replicas=4, multi_client=True)
+
+        def teardown():
+            warps3.cleanup(multi_client=True)
+
+        request.addfinalizer(teardown)
+        return warps3
+
+    def teardown(self):
+        OSDKeyrotation().set_keyrotation_schedule("@weekly")
+
+    def test_cluster_wide_encryption_key_rotation_system(
+        self,
+        teardown_project_factory,
+        pvc_factory,
+        pod_factory,
+        threading_lock,
+        bucket_factory_session,
+        mcg_obj_session,
+        noobaa_db_backup_and_recovery_locally,
+        validate_noobaa_rebuild_system,
+        validate_noobaa_db_backup_recovery_locally_system,
+        warps3,
+    ):
+        """
+        1. Set cluster wide keyrotation period to every 5 mins by editing storage cluster and capture the keys details
```

Comment:
The steps "2, 3 and 4" from actual test case are missing. Please cover them. 
Always copy paste the steps from the actual test case.